plugin/templates/_package.json

```
@@ -33,7 +33,7 @@
     "npm-run-all2": "^6.1.2"
   },
   "engines": {
-    "node": "^18.18.0 || ^20.9.0 || >=21.1.0"
+    "node": "^20.19.0 || ^22.13.0 || >=24"
   },
   "peerDependencies": {
     "eslint": ">=8.57.0"
```

```suggestion
    "eslint": ">=9.0.0"
```

to reflect that the template installs `"eslint": "^9.0.0"` in ln 27 and that ESLint 8 anyway entered end-of-life already more than one year ago on Oct 5, 2024 (see https://eslint.org/version-support/)

Since this PR is a breaking change, it would be a good place to make this update.

---

Makes sense to me! 👍